### ISSUE TITLE
No audio playing

### ISSUE THREAD

**Issue by Fury454** [olegos2/mobox#490]
Whenever I'm playing PC games on the emulator, I can't hear any audio

**Comment by Tattimaan** [comment]
Yes, it is also happening in my s21 fe snapdragon 888 varient with android 14
